### ISSUE TITLE
variables that need to be split up get their comments lost

### ISSUE THREAD

**Issue by lukeapage** [mohebifar/lebab#152]
```
// comment
var a = 1, b = 2;
b = 2;
```
==>
```
const a = 1;
let b = 2;
b = 2;
```

**Comment by nene** [comment]
Thanks for reporting.

**Comment by lukeapage** [comment]
comments between the variables get saved but put in a weird place

```
var a = 1, // comment
      b = 2;
b++;
```
==>
```
const
// comment
a = 1;
let b = 2;
b++;
```

**Comment by nene** [comment]
Fixed the main issue together with #148.

Not really sure what's going on with the weird placement of comment inside VariableDeclaration.

**Comment by nene** [comment]
There's a Recast bug https://github.com/benjamn/recast/issues/191 for this odd comment placement. Apparently this only happens for trailing line-comments.

I guess there's currently not much we can do about this in Lebab.
